stdlib/messaging/nats/src/main/ballerina/src/nats/streaming/listener.bal

```diff
@@ -71,9 +72,24 @@ public type StreamingListener object {
             return detachFromNatsConnection(self, natsConnection);
         }
     }
-
-     function init() = external;
-     function subscribe() = external;
-     function attach(service serviceType, Connection? conn) = external;
-     function detach(service serviceType) = external;
 };
+
+function streamingListenerInit(StreamingListener lis) =
```

Comment:
Do we need the function prefix "streaming" or "basic"? We already have that info in the module name right?

Reply:
Yes. This is because the interop functions are outside the object declarations and so there is an error saying that there are 2 duplicate functions if we use the same name. (Compiler errror: re-declared symbol)